variablecompany/bite
# Feat/add order logic part

## Describe your changes

Adds the cart to the home screen, will be in the menu screen when things on the menu will be completed

<img src="https://user-images.githubusercontent.com/11355609/210270817-c7155bfa-5f01-4c2b-a0ee-9fed847f7bc2.png" width=200/>
<img src="https://user-images.githubusercontent.com/11355609/210270835-8faa708b-11d2-4de6-8054-a1b3453176eb.png" width=200/>

## How to test

you should see the cart in home
- press on view order will take you to the order detail screen
- press on each item will take you to the detail of the item

## Checklist before requesting a review
